protected/models/SurveyLanguageSetting.php

```diff
@@ -92,6 +92,8 @@ public function relations()
     public function rules()
     {
         return [
+            ['surveyls_survey_id','required', 'on' => 'insert'],
+            ['surveyls_language','required', 'on' => 'insert'],
```

Comment:
Rules are checked before survey is created : https://github.com/LimeSurvey/LimeSurvey/blob/develop/protected/controllers/SurveysController.php#L245 
2 solution : 
* we save survey BEFORE languagesettings (tested with : && $survey->save(false) && $languageSetting->validate() not working for language)
* we review languagesettings for 'on'=>'insert'